hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/query/QueryWorkflowImplTest.java

```diff
@@ -69,1059 +69,428 @@
+
+    @Mock(strictness = LENIENT)
+    private QueryChecker checker;
+
+    @Mock(strictness = LENIENT)
+    FileGetInfoHandler handler;
+
+    @Mock(strictness = LENIENT)
+    private QueryDispatcher dispatcher;
+
+    @Mock
+    private HapiOpCounters opCounters;
+
+    @Mock
+    private FeeAccumulator feeAccumulator;
+
+    @Mock
+    private QueryContextImpl queryContext;
+
+    private Query query;
+    private Transaction payment;
+    private TransactionBody txBody;
+    private AccountID payer;
+    private SessionContext ctx;
+    private ByteBuffer requestBuffer;
+
+    private QueryWorkflowImpl workflow;
+
+    @BeforeEach
+    void setup() throws InvalidProtocolBufferException, PreCheckException {
+        when(currentPlatformStatus.get()).thenReturn(PlatformStatus.ACTIVE);
+        when(stateAccessor.apply(any())).thenReturn(new AutoCloseableWrapper<>(state, () -> {}));
+        requestBuffer = ByteBuffer.wrap(new byte[] {1, 2, 3});
+        payment = Transaction.newBuilder().build();
+        final var queryHeader = QueryHeader.newBuilder().payment(payment).build();
+        query = Query.newBuilder()
+                .fileGetInfo(FileGetInfoQuery.newBuilder().header(queryHeader))
+                .build();
+        ctx = new SessionContext();
+
+        payer = AccountID.newBuilder().accountNum(42L).build();
+        final var transactionID = TransactionID.newBuilder().accountID(payer).build();
+        txBody = TransactionBody.newBuilder().transactionID(transactionID).build();
+        when(checker.validateCryptoTransfer(ctx, payment)).thenReturn(txBody);
+
+        when(handler.extractHeader(query)).thenReturn(queryHeader);
+        when(handler.createEmptyResponse(any())).thenAnswer((Answer<Response>) invocation -> {
+            final var header = (ResponseHeader) invocation.getArguments()[0];
+            return Response.newBuilder()
+                    .fileGetInfo(FileGetInfoResponse.newBuilder().header(header).build())
+                    .build();
+        });
+
+        final var responseHeader = ResponseHeader.newBuilder()
+                .responseType(ANSWER_ONLY)
+                .nodeTransactionPrecheckCode(OK)
+                .build();
+        final var fileGetInfo =
+                FileGetInfoResponse.newBuilder().header(responseHeader).build();
+        final var response = Response.newBuilder().fileGetInfo(fileGetInfo).build();
+
+        when(dispatcher.getHandler(query)).thenReturn(handler);
+        when(dispatcher.getResponse(any(), eq(query), eq(responseHeader), eq(queryContext)))
+                .thenReturn(response);
+
+        workflow = new QueryWorkflowImpl(
+                nodeInfo,
+                currentPlatformStatus,
+                stateAccessor,
+                throttleAccumulator,
+                submissionManager,
+                checker,
+                dispatcher,
+                metrics,
+                feeAccumulator,
+                queryContext);
+    }
```

Comment:
same here ?

Reply:
Reverted this to `04712-use-pbj` contents.